packages/hardhat-zksync-solc/src/compile/docker.ts

```diff
@@ -125,7 +120,33 @@ export async function compileWithDocker(
 ) {
     // @ts-ignore
     const dockerInstance: Docker = docker._docker;
-    return await handleCommonErrors(runZksolcContainer(dockerInstance, image, JSON.stringify(input)));
+    return await handleCommonErrors((async () => {
+        const compilerOutput = await runContainer(
+            dockerInstance,
+            image,
+            ['zksolc', '--standard-json'],
+            JSON.stringify(input)
+        );
+        try {
+            return JSON.parse(compilerOutput);
+        } catch {
+            throw pluginError(compilerOutput);
+        }
+    })())
+}
+
+export async function getSolcVersion(docker: HardhatDocker, image: Image) {
```

Comment:
Looks like this function is not needed, probably remove it?

Reply:
I decided to leave it anyway since it might be useful in the future or the user want's to check the solc version before compilation.